### ISSUE TITLE
Not functioning as intended for example experimental data

### ISSUE THREAD

**Issue by 7rebor** [tischi/fiji-plugin-imageRegistration#8]
Example data [here](https://drive.google.com/file/d/1uEhsYwIUq4PVdmelFNtM9R_BZWWZWYCe/view?usp=sharing).

This data has five dimensions, the channels were acquired simultaneously, so don't need any intra-transformations applied. XYZ dimensions differ inter-stack as they were re-assigned at each timepoint (i.e. drift of sample, but also user-defined limits were changed). Time intervals are not consistent (i.e. 1 day, 4 hours then 1 hour x 4), and fluorescence varies over time (not bleaching). Lastly, stack sizes were matched by buffering with blank slices at the end of the stack.

Unfortunately trying to register xyz along time sequence does not improve registration (if anything it is made worse in X, Y and Z).

Could you take a look at the data and see if you can produce a better alignment than the raw data? Maybe I am not using the parameters to their full potential

**Comment by 7rebor** [comment]
I got the plugin working beautifully with my experimental data now

**Comment by tischi** [comment]
@7rebor 

ah, cool!
interesting! 
I just tried it with your data and did not get it too work :-)
I wanted to say that it is due to local deformations (not only shift) and thus the algorithm could not find a match.

which settings did you use for this data set? would you mind sending me a screenshot of the successful settings?

**Comment by 7rebor** [comment]
I used thresholding between 33000, 37000 (the image has odd grayscale values, I have tried to replicate the way I made the stack, but can't find out how...), with 1,1,1 pixel subsampling and using channel 0 as the reference. XY and Z dimensions were registered across time.

I have since removed that file to update it with an [updated version](https://drive.google.com/file/d/1cv5_Sx0Nt0gIF3gSCPxPRRSOra3ZozWm/view?usp=sharing) with the correct grayscale values (not that it should matter) and also in the correct time sequence as I noticed the stacks hadn't been concatenated in the correct way. 

Also, here is [an example](https://drive.google.com/file/d/1I7isTpHQRAosj2CmvuO8XRUIj_405DMl/view?usp=sharing) of what I can currently achieve with an IJ1 macro and a combination of StackReg/MultiStackReg, I use this as my target.
